src/parallel/ocbbs.cpp

```diff
@@ -666,8 +667,15 @@ static double spike_record(void* v) {
 
 static double psolve(void* v) {
 	OcBBS* bbs = (OcBBS*)v;
-	bbs->netpar_solve(chkarg(1, t, 1e9));
-	return 0.;
+	double tstop = chkarg(1, t, 1e9);
+        int enabled = nrncore_is_enabled();
+	if (enabled == 1) {
+		nrncore_psolve(tstop);
+	}else if (enabled == 0) {
+		// Classic case
+		bbs->netpar_solve(tstop);
+	}
+	return double(enabled);
```

Comment:
I didn't get the error code. it should be `return nrncore_psolve(tstop); or return bbs->netpar_solve(tstop);` ?

Reply:
At the moment there is no error code as ```netpar_solve``` is void and fails via ```hoc_execerror```. The return in this case is just an indicator of a normal psolve or a psolve that called coreneuron. Most of these return values for hoc methods are just because a void is not allowed by the hoc language. Errors are generally handled by ```hoc_execerror```. In fact I wonder if, when coreneuron is embedded, all its errors would be steered to the NEURON version of ```hoc_execerror```.